src/game/level.c

```diff
@@ -17,8 +17,8 @@
 
 #include <stdio.h>
 
-#define NUM_PICKUP_OBJ 28
-#define NUM_KILLABLE_OBJ 26
+#define NUM_PICKUP_OBJ 24
+#define NUM_KILLABLE_OBJ 27
```

Comment:
Let's remove this in favor of `NO_ITEM` guard at the end of the list.

Reply:
Ok. Would I change the for loops to something like?
```
            // Calculate number of killable objects in a level
            for (int16_t j = 0; j != NO_ITEM; j++) {
                if (item->object_number == m_KillableObjs[j]) {
                    m_LevelKillables++;
                }
            }
```

Also thanks for marking as draft because I forgot to do that.

Reply:
No, the point of having a guardian is to avoid having to specify the array size.
The loop doesn't look right either.

```c
static int16_t m_KillableObjs[] = { ... };

...

for (int j = 0; m_KillableObjs[j] != NO_ITEM; j++) {
    if (item->object_number == m_KillableObjs[j]) {
        m_LevelKillables++;
    }
}
```

Also, no need for being this specific about the loop variable type.